package.json

```
@@ -46,6 +46,7 @@
     "scripts": {
         "vscode:prepublish": "npm run build && npm run lint",
         "build": "tsc",
+        "prebuild": "cp ./src/assets/* ./out/",
```

Now the prebuild will copy every file from the asset folder into the out folder but only copying the .css seems necessary.

```suggestion
        "prebuild": "cp ./src/assets/*.css ./out/",
```

This can be entirely removed if we change the path in the copyCSSClass function

---

I removed the line